packages/rolldown/tests/cli/__snapshots__/cli-e2e.test.ts.snap

```diff
@@ -59,7 +59,7 @@ OPTIONS
   --inline-dynamic-imports    Inline dynamic imports.
   --input <input>             Entry file.
   --intro <intro>             Code to insert the top of the bundled file (inside the wrapper function).
-  --keep-names                Keep function/class name.
+  --keep-names                Keep function and class names during bundling.
```

Comment:
`during` sounds a little weird, it sounds like it will not be preserved after bundling?

Reply:
Let me tweak it a little bit. It's CC.